sdk/storage/Azure.Storage.Common/tests/Shared/TestExceptionPolicy.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+using Azure.Core.Testing;
+
+namespace Azure.Storage.Common.Test
+{
+    public class TestExceptionPolicy : HttpPipelinePolicy
+    {
+        public List<string> Hosts { get; private set; }
```

Comment:
Nit - Rename to `RequestedHosts` or something to indicate this is a response value not a  parameter

Reply:
Changed to HostsSetInRequests. RequestedHosts made it sound a little like settable config.